common/djangoapps/student/models.py

```diff
@@ -864,6 +864,29 @@ def num_enrolled_in(self, course_id):
 
         return enrollment_number
 
+    def num_enrolled_in_exclude_admins(self, course_id):
+        """
+        Returns the count of active enrollments in a course excluding admins and staff.
+
+        'course_id' is the course_id to return enrollments
+        """
+        # To avoid circular imports.
+        from roles import (CourseInstructorRole, CourseStaffRole)
+        course_locator = course_id
+        if getattr(course_id, 'ccx', None):
+            course_locator = course_id.to_course_locator()
+
+        staff = CourseStaffRole(course_locator).users_with_role()
+        admins = CourseInstructorRole(course_locator).users_with_role()
+
+        enrollment_number = super(CourseEnrollmentManager, self).get_queryset().filter(
+            course_id=course_id,
+            is_active=1,
+        ).exclude(user__in=staff
+        ).exclude(user__in=admins).count()
+
+        return enrollment_number
```

Comment:
Are we counting coaches here? From the title of this PR it seems like that was your intention, but I don't think this code does that.